core/data/src/androidMain/kotlin/io/github/droidkaigi/confsched2023/data/user/DataStoreModule.kt

```diff
@@ -21,6 +21,9 @@ annotation class UserDataStoreQualifier
 @Qualifier
 annotation class SessionCacheDataStoreQualifier
 
+@Qualifier
+annotation class StampDataStoreQualifier
+
```

Comment:
I am not sure this part is really necessary.
However, by including this implementation, we are now able to achieve the intended implementation.